ports/meta/macathena-extra-software-nox/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup		config_package 1.0
+PortGroup		active_variants 1.1
+
+name                macathena-extra-software-nox
+version             0.1
+revision            1
+categories          meta
+maintainers         mit.edu:sipb-macathena
+platforms           darwin
+description         This package pulls in miscellaneous non-Athena-specific software \
+			for a fully-stocked non-graphical Athena machine, like a dialup.
+config.meta
+
+
+depends_lib		port:enscript \
+			port:psutils \
+			port:emacs \
+			port:vim \
+			port:aspell \
+			port:aspell-dict-en \
+			port:screen \
+			port:lynx \
+			port:links \
+			port:rcs \
+			port:cvs \
+			port:git \
+			port:git-cvs \
+			port:subversion \
+			port:bzr \
+			port:mercurial \
+			port:monotone-devel \
+			port:auctex \
+			port:tree \
+			port:py-twisted \
+			port:py-setuptools \
+			port:moreutils \
+			port:socat \
+			port:expect \
+			port:traceroute \
+			port:fortune-mod \
+			port:rlpr \
+			port:texlive \
```

Comment:
Maybe we should move <img src="https://render.githubusercontent.com/render/math?math=\LaTeX"> packages into a variant so that people can opt out of installing them (since they're so large).